pkg/crc/machine/driver_linux.go

```diff
@@ -9,7 +9,7 @@ func init() {
 		Name:          "Libvirt",
 		Platform:      crcos.LINUX,
 		Driver:        "libvirt",
-		UseDNSService: false,
+		UseDNSService: true,
```

Comment:
since this is true, we might as well drop this option as it is always true now.

Reply:
Agreed.